project/Dependencies.scala

```diff
@@ -18,5 +18,5 @@ object Dependencies {
   }
   val jna = "net.java.dev.jna" % "jna" % "4.5.0"
   val jnaPlatform = "net.java.dev.jna" % "jna-platform" % "4.5.0"
-  val appleFileEvents = "com.swoval" % "apple-file-events" % "1.3.2"
+  val swovalFiles = "com.swoval" % "file-tree-views" % "2.0.0-RC2"
```

Comment:
It says RC2. Is it waiting on something?

Reply:
I have been reserving the right to change the public api until this PR is good to go. Moving forward, I'm going to enforce binary compatibility of the library in the 2.x series. As soon as we're ready to merge, I will release a 2.0.0 version and update the relevant commit in this PR.

Reply:
On develop branch I've added -Xfatal-warnings, so could you rebase against it? After that, I am good to go with this PR.

Reply:
Sure. I published swoval v2.0.0, rebased against origin/develop and updated Dependencies.scala.